tutorials/navigation/navigation_using_navigationagents.rst

```diff
@@ -3,145 +3,176 @@
 Using NavigationAgents
 ======================
 
-NavigationsAgents are helper nodes to facilitate common calls to the NavigationServer API
-on behalf of the parent actor node in a more convenient manner for beginners.
+NavigationsAgents are helper nodes that combine functionality 
+for pathfinding, path following and agent avoidance for a Node2D/3D inheriting parent node.
+They facilitate common calls to the NavigationServer API on 
+behalf of the parent actor node in a more convenient manner for beginners.
 
 2D and 3D version of NavigationAgents are available as
 :ref:`NavigationAgent2D<class_NavigationAgent2D>` and
 :ref:`NavigationAgent3D<class_NavigationAgent3D>` respectively.
 
-NavigationsAgents are entirely optional for navigation pathfinding.
-The functionality of NavigationsAgents can be recreated with scripts and direct
-calls to the NavigationServer API. If the default NavigationsAgent does not do what you want
-for your game feel free to design your own NavigationsAgent with scripts.
+New NavigationAgent nodes will automatically join the default navigation map on the World2D/World3D.
 
-.. warning::
-
-    NavigationsAgent nodes and NavigationServer ``agents`` are not the same.
-    The later is an RVO avoidance agent and solely used for avoidance.
-    RVO avoidance agents are not involved in regular pathfinding.
+The functionality of NavigationsAgent nodes can be recreated with scripts and direct
+calls to the NavigationServer API should the default NavigationsAgent 
+nodes not do what is required for a project and specific gameplay.
```

Comment:
@smix8 sorry for tagging you on something what's already merged. Is this sentence phrased as really intended? I dont understand what does it mean circa from the word "should" on second line...

Reply:
It means to role your own agent implementation if the default does not what you need as the NavigationAgent node is optional to work with the navigation system. That block was a copy&paste from the existing doc with not much thought behind. Could be reworded, there are also smaller typos to fix.